linode/manager
# Remove Ada chat bot

## Description

**What does this PR do?**
- Removes the Ada chatbot from Cloud Manager
- Replaces the "Support Bot" tile on the Support Page with a new "Linode Status Page" tile

## How to test

**What are the steps to reproduce the issue or verify the changes?**
Run the app and ensure the following:
- The app runs correctly and there are no regressions introduced by the removal of the Ada script
- The Ada script is not loaded at all 
- All references to Ada code have been removed from the codebase
- The new Support page tile links to status.linode.com and looks correct on mobile
